src/ui/components/EventsCalendarTeaser.tsx

```diff
@@ -35,15 +35,7 @@ export function EventsCalendarTeaser(props: Props) {
           </React.Fragment>
         ))
```

Comment:
There is an issue with this logic, when first loading the screen (with no data) the loading component is not being displayed, it shows the empty state component for a moment and then shows the upcoming events. Can you please check that as well?

Reply:
I fixed it [here](https://github.com/litentry/litentry-app/pull/1030/commits/c2d0a219a3b13772468c8f84404472185ceebb3c)